pycytominer/tests/test_cyto_utils/test_collate.py

```diff
@@ -0,0 +1,190 @@
+import os
+import pytest
+
+import pandas as pd
+from pycytominer.cyto_utils.collate import collate
+
+test_config_location = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "..", "cyto_utils/ingest_config.ini")
+)
+test_data_location = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "test_data/collate")
+)
+test_backend_location = os.path.abspath(
+    os.path.join(
+        os.path.dirname(__file__),
+        "..",
+        "test_data/collate/backend/2021_04_20_Target2/BR00121431/BR00121431.sqlite",
```

Comment:
Can you define this variable first:

`"test_data/collate/backend/2021_04_20_Target2/BR00121431/"`

and then add it to the path with all plate plus extensions?

e.g.

```python
plate = "BR00121431"
backend_plate_dir = pathlib.Path(f"test_data/collate/backend/{batch}/{plate}/{plate}.sqlite")
# and so on...
```

I think it's easier to read and modify this way

Reply:
also, converting to pathlib will help with parsing the `/` on windows systems.